src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -72,14 +69,25 @@ export class FileExplorerComponent implements OnInit {
     });
   }
 
+  sumSize() {
+    this.totalSize = 0;
+    this.selectedFiles.forEach(f => {
+      if (f.data.type === 'File') {
+        this.totalSize += parseFloat(f.data.size);
```

Comment:
Instead of calling `parseFloat()` every time the selection changes, why not store `size` as a `number` and call `parseFloat()` in the code that builds a `FileData`?

Reply:
Good catch, I fixed this issues. Let me know If there's something else I need to change. Thanks!